src/main/common/maths.c

```diff
@@ -360,3 +361,30 @@ int16_t qMultiply(fix12_t q, int16_t input) {
 fix12_t  qConstruct(int16_t num, int16_t den) {
     return (num << 12) / den;
 }
+
+typedef union
+{
+    float    f;
+    uint32_t l;
+} float_long;
+    
+
+#ifdef USE_THRUST_LINEARIZATION
+// see https://en.wikipedia.org/wiki/Fast_inverse_square_root
+float fast_rsqrt(float number)
+{
+    float_long i;
+    float_long y;
+	float x2;
```

Comment:
Consistent indentation missing.

Reply:
It's consistent with a tab width of 4. Isn't that what we're using?

Reply:
The way GitHub renders it shows that something's shifted.

Reply:
Probably tabs and spaces mixed. Betaflight wants only spaces.